src/app/modules/shared-components/components/section-header/section-header.component.ts

```diff
@@ -10,6 +10,9 @@ export class SectionHeaderComponent {
   @Input() title = '';
   @Input() icon = '';
   @Input() theme: 'success' | 'warning' | 'danger' = 'success';
+  @Input() styleClass: string | string[] | Set<string> | { [klass: string]: boolean } = '';
+  @Input() collapsed = true;
+  @Input() collapsible = false;
```

Comment:
To me, section header should not have any collapse-related input because it should not know where or how it is used. But it may offer slots/child content through `ng-content` or template.

Reply:
Fixed